### PR TITLE
feat : Finished Home and product page

### PR DESCRIPTION
**Pull request by Naman-Saxena1**
Finished Basic version of Home and Product page of E-commerce

### CODE REVIEW

File: Pages/Product-Page/product.html [hunk 1]
```diff
@@ -0,0 +1,433 @@
...
+                            <button class="outline-card-secondary-btn">
+                                <i class="fa fa-shopping-cart fa-x" aria-hidden="true" ></i>
+                                &nbsp;Add To Cart
+                            </button>
+                            <button class="card-icon-btn outline-card-secondary-btn"><i class="fa fa-heart-o fa-x" aria-hidden="true"></i></button>
+                        </div>
+                        <div class="badge-on-card">
+                            Manga
+                        </div>
+                        <button class="card-dismiss-btn outline-card-secondary-btn">X</button>
```
Comment (amantulsyan35) [inline Pages/Product-Page/product.html]: In the dismiss button there is some margin to the right side but no margin from the top side, margin on all 4 sides would make it more consistent

Comment (Naman-Saxena1) [reply Pages/Product-Page/product.html]: Good point, plus by mistake I added cards with dismiss but cards with dismiss are not needed on product page, so I will remove those buttons. Thanks for review.

**Comment by varun-Tad** [comment]
Looks great. you can maybe add outline:none to the search box

**Comment by Naman-Saxena1** [comment]
> Looks great. you can maybe add outline:none to the search box

Done, thanks for suggestion.

File: index.html [hunk 2]
```diff
@@ -43,7 +46,68 @@
 
     <div class="homepage-main">
         <div class="homepage-content">
-            <p>This is main</p>
+            <div class="genre-cards-container">
+                <div class="genre-cards"><a href="Pages/Product-Page/product.html"><h2>Fiction</h2></a></div>
+                <div class="genre-cards"><a href="Pages/Product-Page/product.html"><h2>Sci-fi</h2></a></div>
```
Comment (Shivani-1524) [inline index.html:51]: just a suggestion could reduce the size of the font and the genre-cards

**Comment by Shivani-1524** [comment]
Suggestion to add a pointer cursor when user hovers on card in product page.

**Comment by Naman-Saxena1** [comment]
> Suggestion to add a pointer cursor when user hovers on card in product page.

Done. I checked Myntra and Flipkart, they have implemented same as your this suggestion. Thanks for suggestion 😄

File: index.html (hunk 2 above)
Comment (Naman-Saxena1) [reply index.html:51]: Done in next PR. 👍

**Pull request merged**